wazuh/wazuh-kibana-app
# [Module] [Office365] [Panel] Add the view for the module configuration

### Description
This PR adds the module configuration to the `Panel` tab of the Office 365 module.

Features:
  - Created reusable component to display the module configuration
  - Adapted the component to the `Office 365` module use case

### Tests
- For a cluster with at least 2 managers, configure the Office 365 module with different settings for each node
  - Check the module configuration is displayed for each node. An accordion should appear
  
- For a cluster with at least 2 managers, configure the Office 365 module for only one manager
  - Check the module configuration is displayed for the configurated node
  
- For a cluster with at least 2 managers, without Office 365 configurations
  - Should display a message with the configuration is unavailable
  
- For an environment in `manager mode` configure the Office 365 module for the manager
  - Check the module configuration is displayed for the manager

- For an environment in `manager mode` without Office 365 configuration
  - Should display a message with the configuration is unavailable

- On the UI, pin an agent.
  - Should display a message with the configuration is unavailable
  
- When there is some error, should display it using a `EuiCallOut`.

Closes #3539
